--test--/pet.test.js

```diff
@@ -0,0 +1,174 @@
+const Pet = require("../src/pet");
+
+describe("constructor", () => {
+  it("returns an object", () => {
+    expect(new Pet("Fido")).toBeInstanceOf(Object);
+  });
+
+  it("sets the name property", () => {
+    const pet = new Pet("Fido");
+    expect(pet.name).toEqual("Fido");
+  });
+
+  it("has an initial age of 0", () => {
+    const pet = new Pet("Fido");
+    expect(pet.age).toEqual(0);
+  });
+
+  it("has initial hunger of 0", () => {
+    const pet = new Pet("Fido");
+    expect(pet.hunger).toEqual(0);
+  });
+
+  it("set the children property as an empty array", () => {
+    const pet = new Pet("Fido");
+
+    expect(pet.children).toEqual([]);
+  });
+});
+
+describe("growUp", () => {
+  it("increments the age by 1", () => {
+    const pet = new Pet("Fido");
+    pet.growUp();
+    expect(pet.age).toEqual(1);
+  });
+
+  it("has initial fitness of 10", () => {
+    const pet = new Pet("Fido");
+    expect(pet.fitness).toEqual(10);
+  });
+
+  it("returns true if the pet is alive", () => {
+    const pet = new Pet("Fido");
+    this.age = 29;
+    this.hunger = 9;
+    this.fitness = 1;
+    expect(pet.isAlive).toBe(true);
+  });
```

Comment:
This is passing falsely. You can see this for yourself by changing the values to something like age = 300, hunger = 300 and fitness = -300 and running the tests again.
The problem is you are not changing the properties on the pet instance at all, you need to use the `pet.{property}` for this and not `this.{property}`:
```
  it("returns true if the pet is alive", () => {
    const pet = new Pet("Fido");
    pet.age = 29;
    pet.hunger = 9;
    pet.fitness = 1;
    expect(pet.isAlive).toBe(true);
  });
```